domains/dtech.json

```diff
@@ -4,6 +4,6 @@
         "email": "brianpeter930@gmail.com"
     },
     "record": {
-        "TXT": "_github-pages-challenge-pgwiz.dtech"
+        "A": "128.204.223.70"
```

Comment:
IP needs to be in an array

Reply:
also gives error, probably because it isnt connect to a domain. if you want to connect it there then please give us a preview of your website